api/internal/localizer/localizer.go

```diff
@@ -19,6 +21,8 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const defaultChartHome = "charts"
```

Comment:
nit: I think we should put this publicly in `api/types/helmchartargs.go` and use it both here and in the helm plugin itself (under the `plugin/builtin` directory). That way we have the const defined in one place for everything.

Reply:
Done. Thx.